local.env.dist

```diff
@@ -0,0 +1,4 @@
+LOGENTRIES_KEY=
+AWS_ACCESS_KEY=
+AWS_SECRET_KEY=
+S3_BUCKET=
```

Comment:
These are fewer than the list in your ReadMe.

Reply:
the ones in the README are truly required.  For local testing, some are provided in the `docker-compose.yml` and some from the `local.env` (especially the ones that hold sensitive data).